NewForce-Cohort-4/react-nutshell-house-baboonpaw
# Shared News Articles

# Description

Added New Article Button, saved articles for specific user, functioning delete article button.

Fixes # Added articles section.

## Type of change
Please delete options that are not relevant.
- 
# Testing Instructions
•	Pull down the branch. If you have not previously cloned the repo and created your own local api/database.json, you will need to do so.
•	You will need to start your json-server on "-p 8088"
•	Once the app loads in the browser, you should see a login form.
•	Log in with your registered email.
•	Click on Articles tab.
•	You should see an Articles heading and a New Article Button. Click New Article.
•	You should see a form to enter Article title, Synopsis, and URL. Fill out the form.
•	Click save article.
•	You should see your article information appear under the Articles heading.
•	You should see a Delete Article button beneath your article info.
•	Click Delete Article.
•	Your article should disappear.


# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
